site/content/docs/developer-guide/release/plugin-updates.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Releasing plugin updates
+slug: updating-plugins
+weight: 200
+---
+
+When you have a newer version of your plugin, you can update your plugin
+manifest at [krew-index] repository to ship this new version to your users.
+
+This manual operation looks like:
+
+1. Update the `version`, `uri` and `sha256` fields of the plugin manifest file.
+1. [Test plugin installation locally]({{< ref "../installing-locally.md" >}})
+1. Make a pull request to [krew-index] to update the plugin manifest file.
+
+> **Note:** Ideally, the specified `version:` field should match the release tag
+of the plugin. (This helps users and maintainers to easily identify which
+version of the plugin they have installed.)
+
+If you only change `version`, `uri` and `sha256` fields of your plugin manifest,
+your pull request will be automatically approved, tested and merged ([see an
+example](https://github.com/kubernetes-sigs/krew-index/pull/508/files)).
```

Comment:
This link opens the file diff. Maybe the conversation tab is better here? Or even the bot's comment https://github.com/kubernetes-sigs/krew-index/pull/508#issuecomment-589926592

Reply:
Intentional. But can change.